src/core_read.cpp

```diff
@@ -69,7 +69,7 @@ CScript ParseScript(const std::string& s)
             (w->front() == '-' && w->size() > 1 && std::all_of(w->begin()+1, w->end(), ::IsDigit)))
         {
             // Number
-            int64_t n = atoi64(*w);
+            int64_t n = LocaleIndependentAtoi<int64_t>(*w);
```

Comment:
The condition literally checks for the required format for ParseIntegral<>, so why not use that?

Reply:
Sounds like a good idea now that `ParseIntegral` is part of `master` :)

Reply:
Sorry, I was wrong. This is not a refactor.